### PR TITLE
LRDOCS-6854, what's new in search | 7.2 User

### PR DESCRIPTION
**Pull request by rbohl**
https://issues.liferay.com/browse/LRDOCS-6854, let's get this list complete, then I will flesh out the article

### CODE REVIEW

**Comment by lipusz** [comment]
**Started reviewing...** :mag::nerd_face:

File: user/articles/120-search/02-what-new-search.markdown [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+### Custom filter
+
+### Sort
+
+### Search Insights
+
+Search Insights isn't new, but it's improved. In past versions it showed you
+the full query string sent to the search engine, but now it also displays the
+response from the search engine, and an explanation of the score for each
+search hit.
```
Comment (lipusz) [inline user/articles/120-search/02-what-new-search.markdown]: Suggest: if we won't document this at any other places "in detail", I'd also add a note the "Enable Score Explanation" feature is configurable through the widget configuration.
>If this is checked, the response string will include an explanation of how the score was computed for each result returned by the search engine.

File: user/articles/120-search/02-what-new-search.markdown [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+Indexing now displays a progress bar so you can see right in the UI when the
+reindexing action has completed.
+
+<!-- ###Custom Rankings UI? Will it make it in?-->
+
+## New System Settings
+
+Control Panel &rarr; Configuration &rarr; System Settings &rarr; Search
+
+NOTE TO SME: Nothing here. I surveyed the top level headings but didn't dive into each System
```
Comment (lipusz) [inline user/articles/120-search/02-what-new-search.markdown]: Add: well, we've just added `TitleFieldQueryBuilderConfiguration` as part https://issues.liferay.com/browse/LPS-90630 yesterday. :-D

https://issues.liferay.com/browse/LPS-95729 created to add help text for the new properties.
https://issues.liferay.com/browse/LRDOCS-6852 to document the new properties.

File: user/articles/120-search/02-what-new-search.markdown [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# What's New in Search
+
+## New Widgets
+
+### Custom filter
+
+### Sort
```
Comment (lipusz) [inline user/articles/120-search/02-what-new-search.markdown]: Add: Allows users to order results display based upon certain fields.

**Comment by lipusz** [comment]
Hey Russ,

Besides what I've already commented here, there are some items we are missing from the list. I've tried my best to collect everything here: https://grow.liferay.com/learn/Search+and+Enterprise+Search+in+7.2#section-Search+and+Enterprise+Search+in+7.2-Release+Highlights

You can use it a reference to update the doc.

Thank you! @rbohl

File: user/articles/120-search/02-what-new-search.markdown (hunk 1 above)
Comment (rbohl) [reply user/articles/120-search/02-what-new-search.markdown]: We have it documented already, in fact: https://portal.dev.liferay.dev/docs/7-2/user/-/knowledge_base/u/search-insights#explaining-search-results

**Pull request closed without merging**